lib/broccoli/ember-app.js

```diff
@@ -467,6 +464,22 @@ class EmberApp {
     }
   }
 
+  _addJqueryInLegacyEmber() {
+    let emberJQuery = this.project.findAddonByName('@ember/jquery');
+    if (emberJQuery) {
+      this.vendorFiles['jquery.js'] = emberJQuery.paths.jquery;
```

Comment:
If the addon is found, we shouldn't do anything with `vendorFiles` (we should instead rely on the addon having done a normal `app.import`).

Reply:
@rwjblue I know it's an intimate API but some addons can [rely](https://github.com/san650/ember-cli-page-object/blob/ea4bd566089c4ee7f3e5c9a1e629279e3b4ea736/index.js#L23) on `this.vendorFiles['jquery.js']` value. Maybe we can keep it as a getter with a deprecation?

Reply:
Seems fine to me, in that case the addon should check with `@ember/optional-features` first (which would be how they avoid the deprecation).

Reply:
From the EmberObserver [code search](https://emberobserver.com/code-search?codeQuery=%5B%27jquery.js%27%5D) looks like currently "ember-cli-page-object" is the only public addon which needs to detect "jquery" existance at build time. So it potentially could be solved in this addon space. 

However `this.vendorFiles['jquery.js']` was a thing for ages so it might make a sense to leave it readable for some time..